### PR TITLE
Management endpoint

### PR DESCRIPTION
**Pull request by licarth**
See https://github.com/Zoapp/backend/issues/56. Adds `/management` endpoint.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -18,9 +19,10 @@ try {
 }
 
 // merge default and user configuration
-const config = {
-  ...defaultAppConfig,
-  ...userConfig, // overide with userConfig data
-};
+const config = merge(
```
Comment (lodelestra) [inline src/index.js]: What is the advantage of lodash merge over destructuring assignment ?

Comment (licarth) [reply src/index.js]:
```
> let defConf = {a:{b:1, c:1}};
undefined
> let userConf = {a:{c:2}};
undefined
> {...defConf, ...userConf};
{ a: { c: 2 } }
> const _ = require('lodash');
undefined
> _.merge({}, defConf, userConf);
{ a: { b: 1, c: 2 } }
```

File: src/routes/index.js [hunk 2]
```diff
@@ -90,4 +93,14 @@ export default (zoapp) => {
     ["admin", "master", "application"],
     admin.getParameterValue,
   );
+
+  console.log(config.backend.managementEndpoint)
```
Comment (lodelestra) [inline src/routes/index.js]: If this log is made to be kept its must use logger.info IMO

Comment (licarth) [reply src/routes/index.js]: I'll remove that.

**Pull request merged**